extension/chrome/elements/compose-modules/formatters/encrypted-mail-msg-formatter.ts

```diff
@@ -206,8 +206,8 @@ export class EncryptedMsgMailFormatter extends BaseMailFormatter {
     return await SendableMsg.createSMimeEncrypted(this.acctEmail, this.headers(newMsg), encryptedMessage.data, { isDraft: this.isDraft });
   };
 
-  private formatEncryptedMimeDataAsPgpMimeMetaAttachments = (data: Uint8Array) => {
-    const attachments: Attachment[] = [];
+  private formatEncryptedMimeDataAsPgpMimeMetaAttachments = async (data: Uint8Array, pubkeys: PubkeyResult[]) => {
+    const attachments: Attachment[] = await this.view.attachmentsModule.attachment.collectEncryptAttachments(pubkeys);
```

Comment:
As far as I remember, on the top level of a PGP/Mime message there should be only 2 parts: version and the encrypted message. If we need to include the attachment, we should probably include it in the second part (which is a MIME message itself).
As they say, 
```
An advantage to this scheme is that the message structure, like attachment metadata, is not leaked to someone who intercepts the encrypted message... However, to be able to read just the message body, it is necessary to download the whole message, including all attachments, because everything is encrypted together.
```
https://proton.me/support/pgp-mime-pgp-inline